recipes/networkit/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "networkit" %}
+{% set version = "5.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "4fd9439a155cc569000ec039e13680804d8cc233c6d602c88cf0f63174b2babd" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: 'https://files.pythonhosted.org/packages/d4/36/39298b5c1ddccd062505e7e2f9dcb14235ddd158008bda62cdbb907de8e3/networkit-5.0.tar.gz'
+  sha256: '4fd9439a155cc569000ec039e13680804d8cc233c6d602c88cf0f63174b2babd'
+  
+build:
+  skip: True  # [py27 or py33 or win or win_64]
+  number: 0
+  script:
+    - '{{PYTHON}} -m pip install --ignore-installed -vv .'
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.5
+    - ninja >=1.9.0
+  host:
+    - python
```

Comment:
Normally only `python` and `pip` are needed to install a package, all other can be omitted. Please check this.

Reply:
I was having problems in compiling from source without some of the packages (definitely sphinx, cython, openmp, wheel, ipython and jupyter are needed). I have implemented all other suggestions, thank you!

Reply:
Good to know. This then probably needs some work upstream then.

Reply:
Thank you! I will communicate with the developers (I just took charge of making the recipe).